.env.beta

```diff
@@ -42,3 +42,6 @@ REACT_APP_BSC_NATIVE_TO_ERC677_BRIDGE_FOREIGN_ADDRESS="0x61A8287fA7a9f4D10F4699B
 REACT_APP_BSC_FUSE_TOKEN_ADDRESS="0x5857c96DaE9cF8511B08Cb07f85753C472D36Ea3"
 
 REACT_APP_UNDER_MAINTENANCE=false
+
+REACT_APP_HOME_FEE_MANAGER_AMB_NATIVE_TO_ERC20_ADDRESS="0x2cA6C7De1d754F5da6c80ac854D87114941A485a"
```

Comment:
feemanager can be changed. Can you use `bridgeMediator.feeManagerContract()` to get the contract address (also can be zero, which means no fees)?